beta/src/components/Landing/Stats.astro

```diff
@@ -96,33 +96,53 @@ async function getGzipSize() {
   }
 }
 
-const labelMapper = {
-  size: getTextLocalized(translations.Landing.Stats.size, lang),
-  contributors: getTextLocalized(translations.Landing.Stats.contributors, lang),
-  stars: getTextLocalized(translations.Landing.Stats.stars, lang),
-  downloads: getTextLocalized(translations.Landing.Stats.downloads, lang),
-  latest: getTextLocalized(translations.Landing.Stats.latest, lang),
-  license: getTextLocalized(translations.Landing.Stats.license, lang),
+const statsMeta = {
+  size: {
+    label: getTextLocalized(translations.Landing.Stats.size, lang),
+    link: "https://bundlephobia.com/package/effector@latest"
+  },
+  contributors: {
+    label: getTextLocalized(translations.Landing.Stats.contributors, lang),
+    link: "https://github.com/effector/effector/graphs/contributors"
+  },
+  stars: {
+    label: getTextLocalized(translations.Landing.Stats.stars, lang),
+    link: "https://star-history.com/#effector/effector&Date"
+  },
+  downloads: {
+    label: getTextLocalized(translations.Landing.Stats.downloads, lang),
+    link: "https://npmtrends.com/effector"
+  },
+  latest: {
+    label: getTextLocalized(translations.Landing.Stats.latest, lang),
+    link: "https://github.com/effector/effector/releases"
+  },
+  license: {
+    label: getTextLocalized(translations.Landing.Stats.license, lang),
+    link: "https://raw.githubusercontent.com/effector/effector/master/LICENSE"
+  }
 };
+
 const gettingStartedText = getTextLocalized(
   translations.Landing.Stats.StartByAddingEffectorAsDependency,
-  lang,
+  lang
 );
 
 const stats = await Promise.all([getRepoStats(), getNpmStats(), getGzipSize()]).then(
-  ([repo, npm, size]) => ({ ...repo, ...npm, ...size }),
+  ([repo, npm, size]) => ({ ...repo, ...npm, ...size })
 );
 ---
 
 <div class="wrapper">
   <div class="stats">
     <div class="counts">
       {
-        Object.entries(stats).map(([label, counter]) => {
+        Object.entries(stats).map(([name, counter]) => {
           return (
             <div class="stats-card">
               <div class="text-4xl">{counter}</div>
-              <div>{labelMapper[label]}</div>
+              <div>{statsMeta[name].label}</div>
+              <a href={statsMeta[name].link} />
```

Comment:
```suggestion
              <a href={statsMeta[name].link} target="_blank" rel="noreferrer noopener" />
```

Reply:
Why not the whole block wrap in the `a`?

Reply:
```astro
<a href="" target="_blank" rel="..." class="stats-card">
  <div class="text-4xl">{counter}</div>
  <div>{statsMeta[name].label}</div>
  <a href={statsMeta[name].link} />
```

Reply:
Hmm, makes sense 👍 I edited this part